src/predict/api/Dockerfile

```diff
@@ -13,19 +13,19 @@ ENV PYTHONUNBUFFERED 1
 
 # Set environment variables
 ENV FLASK_APP=app.py
-ENV GOOGLE_APPLICATION_CREDENTIALS=AirQo-e37846081569.json
+ENV GOOGLE_APPLICATION_CREDENTIALS=airqo-250220-642e046f4223.json
```

Comment:
Thank you @sserurich 
We however don't need to specify these two in the dockerfile given that we already specify them in the .env file

Reply:
cc @Mnoble-19